windows/README.md

```diff
@@ -111,3 +111,15 @@ Set-Item WSMan:localhost\client\trustedhosts -value *
 You can confirm this ran successfully with, `get-item WSMan:\localhost\Client\TrustedHosts`, which should list an output with a `*` under the Value column.
 
 After running this command you should be able to try again and successfully connect to the remote hosts to gather data.
+
+### Server name cannot be resolved error
+
+Run ping \<subject-ipv4\>. If there's no response, you need to take steps to ensure these servers can communicate with each other.
+
+Ensure the security group of the subject VMs allows incoming connections from the controller machine.
+
+Add the subject machines to the TrustedHosts file of the controller - see [Authentication error](#authentication-error).
```

Comment:
Did you have to run `Set-Item WSMan:localhost\client\trustedhosts -value *` as said on the Authentication Error docs? If so, we can move this command to the packer script and set the value of `trustedhosts` to `*` from the AMI itself.

Reply:
Nice pickup, yes I did. I think that's a great idea.

Reply:
@SamDesmondKing coming back to this again, adding this to packer AMI will set the trustedhosts value for both, controller and subject, instances.
My guess is it will not have any impacts (i.e., even if subject instances are set to * for trustedhosts) and we will be able to run machine stats as we are today, but before I make this change (in infra-templates repo) I'd like to know your views if it can break anything?